Cargo.toml

```diff
@@ -120,11 +120,11 @@ sha2 = "0.10"
 shell-words = "1"
 simplelog = "0.12"
 snafu = "0.8"
-strum = "0.26"
-tabled = "0.10"
+strum = "0.27"
+tabled = "0.15"
```

Comment:
The last time we updated `tabled` it changed the output format for `testsys` and broke automation that called it. I'm concerned about this update and how to test it.

Reply:
Talked offline we'll keep it for now but we need to test validate ami and validate ssm parameters to make sure nothing wacky occurs.